R/capture.R

```diff
@@ -1,51 +1,198 @@
 
```

Comment:
We should use `git diff --check` before pushing to avoid this in the future. Is it possible to enforce it in the prereceive hooks somehow?